inc/namespace.php

```diff
@@ -218,6 +218,9 @@ function get_environment_type() : string {
 	if ( defined( 'HM_ENV_TYPE' ) ) {
 		return HM_ENV_TYPE;
 	}
+	if ( defined( 'WP_ENVIRONMENT_TYPE' ) ) {
+		return WP_ENVIRONMENT_TYPE;
+	}
```

Comment:
Can we flip this? I'd rather have `WP_ENVIRONMENT_TYPE` set to the output of this function.